src/buildings/building.gd

```diff
@@ -34,25 +34,44 @@ var is_deconstructing: bool = false
 var ticks_left_to_delete: int
 
 var is_displaying_info_panel = false
-
-
-enum TYPES {
-	HabBuilding,
-	FoodBuilding,
-	WaterBuilding,
-	AirBuilding,
-	CryoPod,
-	MiningBuilding,
-}
+var bonus_multiplier: float = 1
 
 
 func _ready():
 	set_original_color()
 	build_timer_ui.visible = false
 	TickManager.tick.connect(_on_tick)
+	# TickManager.tick.connect(func(): check_for_adjacency_multiplier(type))
```

Comment:
Do we need this merged in if it's been commented out? Also the lambda you're connecting to the tick signal here seems redundant - `Signal.connect()` expects a callable object, so you could just pass the `check_for_adjacency_multiplier` method as a callable  and check the building type within that method.

Reply:
Yeah we dont need it, I forgot to delete